### PR TITLE
Updated SpatiaLite install instructions to suggest first package from recent Debian-based distributions.

### PR DESCRIPTION
**Pull request by sudomann**
Rephrasing to suit current Debian distros

### CODE REVIEW

File: docs/ref/contrib/gis/install/spatialite.txt [hunk 1]
```diff
@@ -7,9 +7,7 @@ spatial database.
 
 First, check if you can install SpatiaLite from system packages or binaries.
 
-For example, on Debian-based distributions, try to install the
-``spatialite-bin`` package. For distributions that package SpatiaLite 4.2+,
-install ``libsqlite3-mod-spatialite``.
+For example: With Debian-based distributions that package SpatiaLite 4.2+ in their recent releases (e.g Debian Stretch, Ubuntu 18.04, etc.), install ``libsqlite3-mod-spatialite``. For older releases, try to install the ``spatialite-bin`` package.
```
Comment (felixxm) [inline docs/ref/contrib/gis/install/spatialite.txt]: IMO previous wording and sentence construction was correct, so maybe:
```
For example, on Debian-based distributions that package SpatiaLite 4.2+, try to
install the ``libsqlite3-mod-spatialite`` package. For older releases install
``spatialite-bin``.
```
Please wrap docs at 79 chars.

**Comment by felixxm** [comment]
Updated in #11243.

**Comment by sudomann** [comment]
I messed up that pull request didn't I @felixxm ? 
My apologies, I'm very unfamiliar with performing pull requests. I just saw so many other commits attached to mine was wondering how/why.

**Comment by felixxm** [comment]
@sudomann Don't worry :+1: . You sent PR from `stable/2.2.x` to `master` that's why so many commits appeared. I cherry-picked your change.

**Pull request closed without merging**